resilience4j-circuitbreaker/build.gradle

```diff
@@ -1,8 +1,5 @@
-apply plugin: 'jcstress'
-
 dependencies {
     compile project(':resilience4j-core')
```

Comment:
Would you like to migrate `compile` to `api` or `implementation` and `testCompile` to `testImplementation`. We do you have branch to reduce transitive dependencies. Many dependencies are changed to compileOnly in this branch.

Reply:
@RobWin I can try, sure. I suggest doing that in a seperate PR though, because this is not required for updating gradle.

Let me know if that's ok for you, or if I should use this PR.